src/rabbit_epmd_monitor.erl

```diff
@@ -49,10 +51,17 @@
 start_link() ->
     gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
 
+
+epmd_port_please(Mod, Name, Host)->
+  case Mod:port_please(Name, Host) of
+    noport -> erl_epmd:port_please(Name, ?LOCAL_IPV6_ADDRESS);
```

Comment:
Why use `Mod` for the first call, then hard-code `erl_epmd`? I suppose this is a copy-paste error.

Reply:
Also, would it be possible to move this function to a single module, export it and use it everywhere, including `rabbit_networking` and so on? Plus a comment explaining why we need this.

Reply:
> Why use `Mod` for the first call, then hard-code `erl_epmd`? I suppose this is a copy-paste error.

Ops, yes, changed :)! 

>Also, would it be possible to move this function to a single module

Ok done